<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>612</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QDockWidget" name="InputDock">
        <widget class="QWidget" name="dockWidgetContents_2">
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="LatLabel">
            <property name="text">
             <string>Latitude</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="LatLineEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="LongLabel">
            <property name="text">
             <string>Longitude</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="LongLineEdit"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="WSLabel">
            <property name="text">
             <string>Wind Speed</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="WSLineEdit"/>
          </item>
          <item row="5" column="0">
           <widget class="QRadioButton" name="HighRButton">
            <property name="text">
             <string>High Tide</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QRadioButton" name="LowRButton">
            <property name="text">
             <string>Low Tide</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QPushButton" name="ClearPButton">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QPushButton" name="ApplyPButton">
            <property name="text">
             <string>Apply</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDockWidget" name="ImageDock">
        <property name="minimumSize">
         <size>
          <width>192</width>
          <height>350</height>
         </size>
        </property>
        <widget class="QWidget" name="dockWidgetContents">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="ImView">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">Background : rgb(255, 255, 255)</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QDockWidget" name="TableDock">
        <widget class="QWidget" name="dockWidgetContents_3">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QTableWidget" name="TableWidget"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
